CMakeLists.txt

```diff
@@ -98,6 +98,10 @@ if(WITH_CCACHE)
     else()
       set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE} CACHE STRING "ccache executable")
     endif()
+    # See http://peter.eisentraut.org/blog/2014/12/01/ccache-and-clang-part-3/ why that is needed.
+    if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
```

Comment:
Probably we should just set this to true (as is default starting from [ccache 3.3](https://ccache.samba.org/releasenotes.html)) to avoid problems in the future.

Reply:
Agreed. This problem can appear on GCC at any time too.